edgexfoundry/edgex-go
# feat: Consume breaking JWT audience change in go-mod-secrets

Pass default audience of 'edgex' into commands to create Vault roles for JWTs.  This change is needed for integration with the OpenZiti external JWT validator.

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions
"make docker" then in edgex-compose, "make run dev"
Steal the secrets token out of /tmp/edgex/secrets/core-data/secrets-token.json and get a JWT:

curl -ki -H"X-Vault-Token: hvs.CAESblah" http://localhost:8200/v1/identity/oidc/token/core-data
HTTP/1.1 200 OK
...token":"eyJhbGci...

Take the resulting token, paste it into jwt.io or jwt.ms and confirm the "aud" claim is "edgex".


## New Dependency Instructions (If applicable)
